### ISSUE TITLE
Add support for imageBackgroundRemoval() method to work with image URL.

### ISSUE THREAD

**Issue by adilentiq** [Runware/sdk-python#13]
### Description
Currently, imageBackgroundRemoval() doesn't seem to work with an image URL, I'm getting the following error:

```html
OSError: The file at https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png could not be read.
```

### Context
Runware Python SDK version: `runware==0.2.6`

### Requirements
 - Python installed on your workstation

### Reproduction steps

1. Create a directory

2. Create a file e.g `image_background_removal.py` with the following content:
```python
from runware import Runware, IImageBackgroundRemoval
import asyncio
import os
from dotenv import load_dotenv

load_dotenv()


async def main() -> None:
    runware = Runware(
        api_key=os.getenv("RUNWARE_API_KEY"), url=os.getenv("RUNWARE_WSS_URL")
    )

    await runware.connect()

    image_url = 'https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png'
    payload = IImageBackgroundRemoval(inputImage=image_url)

    print(f"Payload: {payload}")

    result_image = await runware.imageBackgroundRemoval(payload)

    print(f"Response : {result_image}")


asyncio.run(main())

```

3. Create a virtual environment
  ```bash
  python -m venv venv
  ```

4. Activate venv environment
  ```bash
  # Windows
  venv\Scripts\activate

  # Linux/MacOS
   source venv/bin/activate
  ```

5. Create a requirements.txt file with the following content:
```
aiofiles==23.2.1
asyncio==3.4.3
python-dotenv==1.0.1
runware==0.2.6
websockets==12.0
```

6. Install project dependencies:

```bash
 pip install -r requirements.txt
 ```
 
 7. Create `.env` file and set up environment variables
   ```bash
   touch .env
   source .env
  ```
   
   Example
   ```
   RUNWARE_API_KEY="your_runware_api_key"
  RUNWARE_WSS_URL="wss://ws-api.runware.ai/v1"
  RUNWARE_MODEL="runware:100@1"
  ```
  
  8. Run the python script
  ```bash
  python my_test.py
  ```

### Actual result

```bash
$ python ./image_background_removal.py 
Payload: IImageBackgroundRemoval(inputImage='https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png', includeCost=False, outputType=None, outputFormat=None, rgba=[], postProcessMask=False, returnOnlyMask=False, alphaMatting=False, alphaMattingForegroundThreshold=None, alphaMattingBackgroundThreshold=None, alphaMattingErodeSize=None)       
Traceback (most recent call last):
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\utils.py", line 277, in fileToBase64
    async with aiofiles.open(file_path, "rb") as file:
  File "C:\Users\andre\playground\venv\Lib\site-packages\aiofiles\base.py", line 98, in __aenter__
    self._obj = await self._coro
                ^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\aiofiles\threadpool\__init__.py", line 94, in _open
    f = yield from loop.run_in_executor(executor, cb)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\AppData\Local\Programs\Python\Python312\Lib\concurrent\futures\thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: [Errno 22] Invalid argument: 'https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\andre\playground\image_background_removal.py", line 26, in <module>
    asyncio.run(main())
  File "C:\Users\andre\AppData\Local\Programs\Python\Python312\Lib\asyncio\runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\andre\AppData\Local\Programs\Python\Python312\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\AppData\Local\Programs\Python\Python312\Lib\asyncio\base_events.py", line 687, in run_until_complete  
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\image_background_removal.py", line 21, in main
    result_image = await runware.imageBackgroundRemoval(payload)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 416, in imageBackgroundRemoval
    raise e
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 412, in imageBackgroundRemoval
    return await asyncRetry(
           ^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 50, in asyncRetry
    raise error
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 43, in asyncRetry
    return await apiCall()
           ^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 423, in _removeImageBackground
    image_uploaded = await self.uploadImage(inputImage)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 661, in uploadImage
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 659, in uploadImage
    return await asyncRetry(lambda: self._uploadImage(file))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 50, in asyncRetry
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 659, in uploadImage
    return await asyncRetry(lambda: self._uploadImage(file))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 50, in asyncRetry
    raise error
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 43, in asyncRetry
    return await apiCall()
           ^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 673, in _uploadImage
    image_base64 = await fileToBase64(file) if isinstance(file, str) else file
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\utils.py", line 285, in fileToBase64
    raise IOError(f"The file at {file_path} could not be read.")
OSError: The file at https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png could not be read.     
(venv)
andre@mylaptop MINGW64 C:/Users/andre/AppData/Local/Programs/Microsoft VS Code
$ https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.pnghttps://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png


  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 659, in uploadImage
    return await asyncRetry(lambda: self._uploadImage(file))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 50, in asyncRetry
    raise error
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 43, in asyncRetry
    return await apiCall()
           ^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 673, in _uploadImage
    image_base64 = await fileToBase64(file) if isinstance(file, str) else file
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\utils.py", line 285, in fileToBase64
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 659, in uploadImage
    return await asyncRetry(lambda: self._uploadImage(file))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 50, in asyncRetry
    raise error
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 43, in asyncRetry
    return await apiCall()
           ^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 659, in uploadImage
    return await asyncRetry(lambda: self._uploadImage(file))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 50, in asyncRetry
    raise error
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 43, in asyncRetry
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 50, in asyncRetry
    raise error
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 43, in asyncRetry
    raise error
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\async_retry.py", line 43, in asyncRetry
    return await apiCall()
           ^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\base.py", line 673, in _uploadImage
    image_base64 = await fileToBase64(file) if isinstance(file, str) else file
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\andre\playground\venv\Lib\site-packages\runware\utils.py", line 285, in fileToBase64
    raise IOError(f"The file at {file_path} could not be read.")
OSError: The file at https://upload.wikimedia.org/wikipedia/commons/4/47/PNG_transparency_demonstration_1.png could not be read.     
(venv)
```

### Expected result
As a user I want to be able to call imageBackgroundRemoval() with inputImage parameter: an URL pointing to an image.

**Comment by andreidavid** [comment]
Hi @adilentiq, thank you for the bug report. Should be fixed now in version v0.2.7 :)

**Comment by adilentiq** [comment]
@andreidavid thank you, I confirm that the issue was fixed in v0.2.7.

Payload:
```json
{'inputImage': 'https://raw.githubusercontent.com/adilentiq/test-images/main/image-bmp.bmp', 'includeCost': True, 'outputType': 'URL', 'outputFormat': 'JPG', 'rgba': [], 'postProcessMask': False, 'returnOnlyMask': False, 'alphaMatting': False, 'alphaMattingForegroundThreshold': None, 'alphaMattingBackgroundThreshold': None, 'alphaMattingErodeSize': None}
```

Response:
```json
[IImage(taskType='imageBackgroundRemoval', imageUUID='4f94db42-d22e-4a2f-ad29-21395ad8399a', taskUUID='ba1d8ecb-0efd-4d50-aadb-56e5f61240a9', inputImageUUID='34962cb9-1e09-4d3f-a790-2a6321c751f4', imageURL='https://im.picfinder.dev/image/ws/0.5/ii/4f94db42-d22e-4a2f-ad29-21395ad8399a.jpg', imageBase64Data=None, imageDataURI=None, NSFWContent=None, cost=0.0006)]
```
